build_interface/settings.py

```diff
@@ -44,7 +45,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
```

Comment:
:facepalm:

Reply:
Haha, but it was because this was set to True, I was able to debug art-dash quickly, initially. At that time I didn't know how to set it up locally.